main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -879,8 +879,8 @@ object Parser2 {
 
     private def moduleDecl(mark: Mark.Opened, nestingLevel: Int = 0)(implicit s: State): Mark.Closed = {
       assert(at(TokenKind.KeywordMod))
-      expect(TokenKind.KeywordMod, SyntacticContext.Decl.Module)
-      name(NAME_MODULE, allowQualified = true, context = SyntacticContext.Decl.Module)
+      expect(TokenKind.KeywordMod, SyntacticContext.Keyword)
```

Comment:
Did you change it to use SyntacticContext everywhere a TokenKind.Keyword is used?

Reply:
i changed it to use `SyntacticContext.Keyword` for expect denoting the start of a declaration, and `SyntacticContext.Name` for every name that follows. Potential other cases of `expect` on `TokenKind.Keyword`, I didn't touch. But perhaps I should?

Reply:
You should be consistent

Reply:
I'll go over it again to check for other `expect` with TokenKind.Keyword and name and make sure it's consistent :)